polls/models.py

```diff
@@ -27,3 +28,14 @@ class Choice(models.Model):
     def __str__(self):
         return self.choice_text
 
+
+class UserProfile(AbstractUser):
+    SEX = ((1, 'Male'), (2, 'Female'))
```

Comment:
właśnie tutaj opisujesz to taki sposób, a w formularzu w inny, to nie jest dobrze, stworzenie formularza przez model to rozwiązuje i właśnie zrób te choisy tak jak pisałem wyżej